scripts/build_electricity_demand.py

```diff
@@ -276,7 +293,14 @@ def manual_adjustment(load, fn_load, countries):
 
     time_shift = snakemake.params.load["fill_gaps"]["time_shift_for_large_gaps"]
 
-    load = load_timeseries(snakemake.input.reported, years, countries)
+    if snakemake.params.load["source"] == "tyndp":
+        planning_horizons = int(snakemake.wildcards.planning_horizons)
+    else:
+        planning_horizons = None
+
+    load = load_timeseries(
+        snakemake.input.reported, years, countries, planning_horizons=planning_horizons
```

Comment:
If you are handing the planning horizon explicitly I would prefer doing the same for the other parameters too
```suggestion
        fn=snakemake.input.reported,
        years=years,
        countries=countries,
        planning_horizons=planning_horizons,
```